OpenRA.Mods.Common/Warheads/CreateEffectWarhead.cs

```diff
@@ -19,7 +19,10 @@ namespace OpenRA.Mods.Common.Warheads
 	public class CreateEffectWarhead : Warhead
 	{
 		[Desc("List of explosion effects that can be used.")]
-		public readonly string[] Explosions = new string[0];
+		[SequenceReference("ExplosionCollection")] public readonly string[] Explosions = new string[0];
+
+		[Desc("Explosion collection containing effect.")]
+		public readonly string ExplosionCollection = "explosion";
```

Comment:
Well, Parachutable and WithBuildingExplosion already did.

The main reason is that `Image` would be somewhat unfitting given how the explosion sequences are set up.